lib/pure/htmlgen.nim

```diff
@@ -613,6 +613,10 @@ macro wbr*(e: varargs[untyped]): untyped =
   ## Generates the HTML `wbr` element.
   result = xmlCheckedTag(e, "wbr", commonAttr, "", true)
 
+macro portal*(e: varargs[untyped]): untyped =
+  ## Generates the HTML `portal` element.
```

Comment:
@juancarlospaco please add a url in followup PR (you can batch similar fixes in 1 PR); not everything deserves a url but for things like this IMO it helps